template/package.json

```diff
@@ -75,6 +79,8 @@
     "chromedriver": "^2.21.2",
     "cross-spawn": "^4.0.2",
     "nightwatch": "^0.9.8",
+    "node-sass": "^3.10.1",
```

Comment:
These sass deps will only get loaded if you enable e2e testing. Move it higher up.